lib/perl/Genome/Model/Tools/LiftOver.pm

```diff
@@ -100,10 +102,16 @@ sub execute {
 
     unless( defined $chain_file ) {
         if( $lift_direction eq 'hg19ToHg18' ) {
-            $chain_file = "/gscmnt/gc12001/info/model_data/chain_files/hg19Tohg18/hg19ToHg18.over.chain";
+            my $alloc = Genome::Disk::Allocation->get(allocation_path => 'model_data/chain_files/hg19Tohg18');
+            $self->fatal_message("Couldn't find chain file for hg19Tohg18") unless $alloc;
+
+            $chain_file = File::Spec->join($alloc->absolute_path, "hg19ToHg18.over.chain");
         }
         elsif( $lift_direction eq 'hg18ToHg19' ) {
-            $chain_file = "/gscmnt/gc12001/info/model_data/chain_files/hg18Tohg19/hg18ToHg19.over.chain";
+            my $alloc = Genome::Disk::Allocation->get(allocation_path => 'model_data/chain_files/hg18Tohg19');
```

Comment:
I think we could avoid some code duplication here by just setting a `$allocation_path` variable and doing the the rest after the else block

Reply:
The allocation path and the filename are both different in each case.  However, with a little lowercasing hack I've combined the two.